docker/supervisor-celery.conf

```diff
@@ -7,17 +7,18 @@ logfile=/code/logs/supervisord-celery.log
 # https://github.com/mozilla-services/puppet-config/blob/master/amo/modules/celery/manifests/service.pp#L20
 # Changes:
 # - Set concurrency (number of workers) back to the default
-# - The code auto-reloads for development
-command=celery -A olympia.amo.celery:app worker --autoreload -E --loglevel=INFO
+command=watchmedo auto-restart -d /code/ -p '*.py' -R celery -- -A olympia.amo.celery:app worker -E --loglevel=INFO
```

Comment:
This looks like the right thing to do but we have [had issues](https://forums.docker.com/t/file-access-in-mounted-volumes-extremely-slow-cpu-bound/8076) with file watching in the past. Just something to be aware of.